test/Spring-test.js

```diff
@@ -391,4 +391,239 @@ describe('Spring', () => {
       },
     ]);
   });
+
+  it('should NOT warn when parent kills the child Spring', () => {
+    let count = [];
+    let shouldKill = false;
+    const App = React.createClass({
+      propTypes: {
+        kill: PropTypes.func.isRequired,
+      },
+
+      endValue() {
+        if (shouldKill) {
+          this.props.kill();
+        }
+
+        return {val: 400};
+      },
+      render() {
+        return (
+          <Spring defaultValue={{val: 0}} endValue={this.endValue}>
+            {({val}) => {
+              count.push(val);
+              return <div />;
+            }}
+          </Spring>
+        );
+      },
+    });
+
+    let Parent = React.createClass({
+      getInitialState() {
+        return {
+          isAlive: true,
+        };
+      },
+      render() {
+        if (shouldKill) {
+          expect(this.state.isAlive).toBe(false);
+        }
+        return this.state.isAlive && <App kill={() => this.setState({isAlive: false})}/>;
+      },
+    });
+
+    TestUtils.renderIntoDocument(<Parent />);
+
+    expect(count).toEqual([0]);
+    mockRaf.manySteps(4);
+    shouldKill = false;
```

Comment:
This is just testing for that one edge case of rendering something that has been unmounted.